app/controllers/support_interface/provider_test_data_controller.rb

```diff
@@ -0,0 +1,31 @@
+module SupportInterface
+  class ProviderTestDataController < SupportInterfaceController
+    before_action :check_this_is_sandbox
+
+    DEFAULT_APPLICATION_COUNT = 100
+    DEFAULT_COURSES_COUNT = 1
+
+    def create
+      GenerateTestApplicationsForProvider.new(
+        provider: provider,
+        courses_per_application: DEFAULT_COURSES_COUNT,
+        count: DEFAULT_APPLICATION_COUNT,
+      ).call
+
+      flash[:success] = 'Scheduled a job to generate test applications - this might take a while!'
+      redirect_to support_interface_provider_applications_path(provider)
+    end
+
+  private
+
+    def check_this_is_sandbox
+      return if HostingEnvironment.sandbox_mode?
+
+      redirect_to support_interface_provider_applications_path(provider)
```

Comment:
V Minor: Do we want to `render_403` or flash a warning here if we're not in Sandbox?